.pre-commit-config.yaml

```diff
@@ -0,0 +1,59 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+    -   id: check-added-large-files
+        name: 🐘 Check for added large files
+    -   id: check-toml
+        name: ✔️  Check TOML
+    -   id: check-yaml
+        name: ✔️  Check YAML
+        args:
+        -   --unsafe
+    -   id: end-of-file-fixer
+        name: 🪚 Fix end of files
+    -   id: trailing-whitespace
+        name: ✂️  Trim trailing whitespaces
+-   repo: https://github.com/asottile/pyupgrade
+    rev: v2.37.3
+    hooks:
+    -   id: pyupgrade
+        name: ⏫ Running pyupgrade
+        args:
+        - --py3-plus
+        - --keep-runtime-typing
+-   repo: https://github.com/myint/autoflake
+    rev: v1.5.3
+    hooks:
+    -   id: autoflake
+        name: ❄️  Running autoflake
+        args:
+        - --recursive
+        - --in-place
+        - --remove-all-unused-imports
+        - --remove-unused-variables
+        - --expand-star-imports
+        - --exclude
+        - __init__.py
+        - --remove-duplicate-keys
+-   repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+    -   id: isort
+        name: 🔄 Formatting imports with isort (python)
+    # -   id: isort
+    #     name: isort (cython)
+    #     types: [cython]
+    # -   id: isort
+    #     name: isort (pyi)
+    #     types: [pyi]
+# -   repo: https://github.com/psf/black
+#     rev: 22.8.0
+#     hooks:
+#     -   id: black
```

Comment:
(or autoflake is good enough?)

Reply:
I put it there in case I wanted to give it a try. Black works well, but I am not specially a big advocate, only used it in small projects and not fully convinced. I feel like it would take some time to properly configure and block rules we don't want to have something nice 
As long as the all the devs in the codebase understand each other code I don't think that's really necessary

I will remove the comments

Reply:
Autoflake and isort (to sort imports automatically) are enough imo